lib/mongodb.js

```diff
@@ -2199,3 +2202,19 @@ function hasDataType(dataType, propertyDef) {
     propertyDef.mongodb.dataType &&
     propertyDef.mongodb.dataType.toLowerCase() === dataType.toLowerCase();
 }
+/**
+* A utility function which adds the connector options to mongodb options
+* the given data type
+* @param {*} commandOptions The command options
+* @param {*} connectorOptions The connector options
+* @param {*} property Connector's option's property name
```

Comment:
I found the param names are confusing, especially the third one. Could you rename them and add an example with `@example` tag ?